Formula/rogue.rb

```diff
@@ -4,6 +4,7 @@ class Rogue < Formula
   homepage "https://sourceforge.net/projects/roguelike/"
   url "https://src.fedoraproject.org/repo/pkgs/rogue/rogue5.4.4-src.tar.gz/033288f46444b06814c81ea69d96e075/rogue5.4.4-src.tar.gz"
   sha256 "7d37a61fc098bda0e6fac30799da347294067e8e079e4b40d6c781468e08e8a1"
+  revision 1
```

Comment:
Wait, why the revision bump? Doesn't `ncurses_define` do exactly what the line you replaced it with does?

Reply:
I removed it. I initially wanted to remove the flag, but it does not build without it. So I am finally just migrating the flag over.